### ISSUE TITLE
Windows api issue, error 127

### ISSUE THREAD

**Issue by BuvinJT** [bytedeco/javacpp#774]
I have a C++ library which needs to leverage the Windows API.  The code works fine naturally in C++ (or from within a Python wrapper over that..).  When wrapped by JavaCPP, however, it all compiles, but an error is produced at runtime.  I am not coming up with a solution despite a great deal of effort.

Here are some snippets of pertinent code:

```
#include <windows.h>
#include <stdio.h>
#include <tchar.h>
#include <psapi.h>
#include <aclapi.h>
#pragma comment(lib, "advapi32.lib")

...

        // Create an "Everyone SID" (security identifier)
        SID_IDENTIFIER_AUTHORITY SIDAuthWorld( SECURITY_WORLD_SID_AUTHORITY );
        const BYTE subAuthCount( 1 );
        const bool isSid( AllocateAndInitializeSid( &SIDAuthWorld, subAuthCount,
            SECURITY_WORLD_RID, 0, 0, 0, 0, 0, 0, 0, &everyoneSid_ ) );

```

After invoking `AllocateAndInitializeSid`,  `GetLastError()` returns 127, which means "The specified procedure could not be found."

I have tried linking / loading advapi32 in various ways.  All I manage to do is break the compilation.  This is a show stopper for me.

**Comment by BuvinJT** [comment]
Fixed!  Turns out I was wrong about the timing of when the error code was tripped.  The above code (calling AllocateAndInitializeSid) executes very early during the instantiation of the first object I was creating from my C++ library with a Java counterpart, and when an error is detected during that, it effectively renders the object unusable.  But... I found that this error 127 was actually set BEFORE ANY OTHER CODE of mine was executed!  I'm 98% sure of this.  Nothing like this happen in any other contexts I've tested (straight C++, Python wrapped, on Linux...), it seems like something with JavaCPP set that off?  So, using a little trickery in my constructor's initializer list I reset the error as the first step for creating it.  Problem solved.  I can't conclusively state this wasn't somehow my fault, but if anyone runs into this problem perhaps this post can save you some headaches.
